run/logging-manual/README.md

```diff
@@ -55,11 +55,13 @@ test/runner.sh sleep 20
 
 ## Environment Variables (Testing)
 
-* `BASE_URL`: Used to designate the URL of the Cloud Run service under system test. The URL is not deterministic, so the options are to record it at deploy time or use the Cloud Run API to retrieve the value.
-* `ID_TOKEN`: Used for authenticated HTTP requests to the Cloud Run service.
+* `BASE_URL`: Specifies the Cloud Run service URL for end-to-end tests.
```

Comment:
The wording for this section seems inconsistent (`BASE_URL` vs `REGION` descriptions for example).
All env vars specify or configure something. 

Can we just say:
- `BASE_URL`: The Cloud Run service URL...
...
- `SERVICE_NAME`: The name of the deployed service. Used in some API calls...